db/schema.rb

```diff
@@ -461,8 +461,6 @@
     t.datetime "updated_at",                                 null: false
   end
 
-  add_index "fee_coverage_detail_bases", ["fee_era_id"], name: "index_fee_coverage_detail_bases_on_fee_era_id", using: :btree
```

Comment:
What happened to those lines?

Reply:
hmmmm, something that should not have happened

Reply:
It's corrected now, please review!